scripts/version.mjs

```diff
@@ -0,0 +1,4 @@
+import versionWriter from '@aurodesignsystem/auro-library/scripts/build/versionWriter.js';
+
+versionWriter.writeDepVersionFile('@aurodesignsystem/auro-dropdown');
+versionWriter.writeDepVersionFile('@aurodesignsystem/auro-input');
```

Comment:
There is no issue with this code, but I made a note here https://github.com/AlaskaAirlines/WC-Generator/discussions/505#discussioncomment-8701632 discussing a potential issue with this versioning process.